ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -154,6 +154,25 @@
 {%     endif %}
 
 ### Router BGP EVPN Address Family
+{%     if evpn_hostflap_detection.window is arista.avd.defined or evpn_hostflap_detection.threshold is arista.avd.defined or  evpn_hostflap_detection.enabled is arista.avd.defined(false) %}
```

Comment:
device documentation should only look at structured_config variables. So we need to adjust this documentation to use the same vars as you use in the eos config template. I can help if you like.